### DISCUSSION TITLE
PowerDNS with Windows and Active Directoy

### DISCUSSION THREAD

**Discussion by mh-muc** [PowerDNS/pdns#14155] [Q&A]
HI,

i want to use PowerDNS in a mixed environment, also including Windows and Active Directory.
PowerDNS is completely responsible for the domain, so no Windows DNS.

The Problem is, i cannot get dynamic updates to work, which is necessary for Windows AD servers.

PowerDNS auth does not serve queries directly.
PowerDNS recursor cannot deal dynamic updates.
In Windows i can only set a DNS server (which will need to be able to deal with queries AND dynamic updates).

Did i miss something or can't PowerDNS be used in such an environment? 
How are dynamic updates supposed to work if PowerDNS auth without recursor cannot deal with queries, but recursor cannot deal with dynamic updates?

**Comment by mh-muc** [comment]
Found it by myself, dnsdist in front...

**Reply by skeels78** [reply]
What does your config look like?

**Reply by eahammack** [reply]
I am about to experiment with this myself. Can you tell us how it worked out for you?

**Comment by SkySai1** [comment]
> HI,
> 
> i want to use PowerDNS in a mixed environment, also including Windows and Active Directory. PowerDNS is completely responsible for the domain, so no Windows DNS.

Hi, can you please send some examples of your settings?
I wanna try to connect pdns to my AD domain
